integration/resource_lacework_alert_profile_test.go

```diff
@@ -32,10 +32,10 @@ func TestAlertProfileCreate(t *testing.T) {
 	actualDescription := terraform.Output(t, terraformOptions, "alert_description")
 
 	assert.Equal(t, "LW_CFG_GCP_DEFAULT_PROFILE", createProps.Data.Extends)
```

Comment:
I would change this test to find the alert by name. There will always be an alert named `Violation`, you can assert that. This way the test will not fail if a new alert is added.